autoload/OmniSharp/actions/usings.vim

```diff
@@ -4,14 +4,18 @@ set cpoptions&vim
 " Accepts a Funcref callback argument, to be called after the response is
 " returned (synchronously or asynchronously) with the number of ambiguous usings
 function! OmniSharp#actions#usings#Fix(...) abort
-  let opts = a:0 && a:1 isnot 0 ? { 'Callback': a:1 } : {}
+  if a:0 && a:1 isnot 0
+    let Callback = a:1
+  else
+    let Callback = function('s:CBFixUsings')
+  endif
+
   if g:OmniSharp_server_stdio
-    let Callback = function('s:CBFixUsings', [opts])
-    call s:StdioFix(Callback)
+    call s:StdioFind(Callback)
```

Comment:
This should be `s/s:StdioFix`, not `s:StdioFind`

Reply:
Done.